lib/matplotlib/cbook/__init__.py

```diff
@@ -13,6 +13,12 @@
 from six.moves import xrange, zip
 from itertools import repeat
 import collections
+
+try:
+    from collections.abc import MappingView
+except ImportError:
+    MappingView = None
```

Comment:
So - `collections.MappingView` is present in all Python versions we support, but `collections.abc.MappingView` is present only for the Python 3 versions:

```
Changed in version 3.3: Moved Collections Abstract Base Classes to the collections.abc module. For backwards compatibility, they continue to be visible in this module as well.
```

https://docs.python.org/3.5/library/collections.html#module-collections

We could use `collections.MappingView` directly, they haven't been deprecated, I guess.